src/views/dialogs/DisplaySettingsDialog.tsx

```diff
@@ -0,0 +1,199 @@
+
+import { layoutMixins } from '@/styles/layoutMixins';
+import { Themes } from '@/styles/themes';
+
+import { STRING_KEYS } from '@/constants/localization';
+
+import { Dialog } from '@/components/Dialog';
+import { Icon, IconName } from '@/components/Icon';
+import { HorizontalSeparatorFiller } from '@/components/Separator';
+
+type ElementProps = {
+  setIsOpen: (open: boolean) => void;
+};
+
+export const DisplaySettingsDialog = ({ setIsOpen }: ElementProps) => {
+  const dispatch = useDispatch();
+  const stringGetter = useStringGetter();
+
+  const currentTheme: AppTheme = useSelector(getAppTheme);
+
+  const sectionHeader = (heading: string) => {
+    return (
+      <Styled.Header>
+        {heading}
+        <HorizontalSeparatorFiller />
+      </Styled.Header>
+    );
+  };
+
+  const themePanels = () => {
+    return (
+      <Styled.AppThemeRoot value={currentTheme}>
+        {[
+          {
+            theme: AppTheme.Classic,
+            label: STRING_KEYS.CLASSIC_DARK,
+          },
+          {
+            theme: AppTheme.Dark,
+            label: STRING_KEYS.DARK,
+          },
+          {
+            theme: AppTheme.Light,
+            label: STRING_KEYS.LIGHT,
+          },
+        ].map(({ theme, label }) => (
+          <Styled.AppThemeItem
+            key={theme}
+            value={theme}
+            backgroundcolor={Themes[theme].layer2}
+            gridcolor={Themes[theme].borderDefault}
+            onClick={() => {
+              dispatch(setAppTheme(theme));
+            }}
+          >
+            <Styled.AppThemeHeader textcolor={Themes[theme].textPrimary}>
+              {stringGetter({ key: label })}
+            </Styled.AppThemeHeader>
+            <Styled.Image src="/chart-bars.svg" />
+            <Styled.CheckIndicator>
+              <Styled.CheckIcon iconName={IconName.Check} />
+            </Styled.CheckIndicator>
+          </Styled.AppThemeItem>
+        ))}
+      </Styled.AppThemeRoot>
+    );
+  };
+
+  return (
+    <Dialog
+      isOpen
+      setIsOpen={setIsOpen}
+      title={stringGetter({ key: STRING_KEYS.DISPLAY_SETTINGS })}
+    >
+      <Styled.Section>
+        {sectionHeader(stringGetter({ key: STRING_KEYS.THEME }))}
+        {themePanels()}
+      </Styled.Section>
+    </Dialog>
+  );
+};
+
+const Styled: Record<string, AnyStyledComponent> = {};
+
+const gridStyle = css`
+  display: grid;
+  gap: 1.5rem;
+`;
+
+Styled.Section = styled.div`
+  ${gridStyle}
+  padding: 1rem 0;
+`;
+
+Styled.Header = styled.header`
+  ${layoutMixins.inlineRow}
+`;
+
+Styled.AppThemeRoot = styled(Root)`
+  ${gridStyle}
+  grid-template-columns: 1fr 1fr;
+`;
+
+Styled.Item = styled(Item)`
+  --border-color: var(--color-border);
+  --item-padding: 0.75rem;
+
+  &[data-state='checked'] {
+    --border-color: var(--color-accent);
+  }
+
+  border: solid var(--border-width) var(--border-color);
+  border-radius: 0.875rem;
+  padding: var(--item-padding);
+`;
```

Comment:
Can we add these rules directly into `Styled.AppThemeItem`.

`Styled.AppThemeItem = styled(Item)` instead of inheriting from `Styled.Item`

Reply:
+1 since Styled.Item seems not used else where

Reply:
ah sorry if it was unclear - pulled this out because I was using it later for the Green / Red is up button preferences in a follow up PR (i.e. [here](https://github.com/dydxprotocol/v4-web/pull/261/files#diff-276e7f8eaf76192edf9910a7de5b47c670b820c1e855561213a7bbb264d22c5cR183))